src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -87,6 +88,8 @@ describe('ReactChildren', function() {
     var mappedChildren = ReactChildren.map(instance.props.children, callback);
     expect(callback).toHaveBeenCalledWith(simpleKid, 0);
     expect(nthChild(mappedChildren, 0)).toBe(simpleKid);
+
+    expect(console.error.calls.length).toBe(1);
```

Comment:
I will fix it soon. Thanks for the feedback.

There are some commits on upstream/master branch, what should I do on my fix branch ? should I fetch and merge those commits ?